packages/transaction-controller/src/TransactionController.ts

```diff
@@ -718,6 +725,7 @@ export class TransactionController extends BaseController<
       hash,
       actionId,
       originalGasEstimate: transactionMeta.transaction.gas,
+      type: TransactionType.retry,
```

Comment:
Just for audit sake, can I confirm we're not also updating the type in the stop method as we're not storing the meta?